src/components/autocompleter/autocompleter.jsx

```diff
@@ -0,0 +1,106 @@
+import PropTypes from 'prop-types';
+import React, { forwardRef, useEffect, useState } from 'react';
+import useFetch from '@bloodyaugust/use-fetch';
+import Select from '../select/select.jsx';
+import ListOption from '../list-option/list-option.jsx';
+import mockConstants from '../../mocks/mock-constants.js';
+
+const { API_ROOT } = mockConstants;
+
+const Autocompleter = forwardRef(function Autocompleter(props, ref) {
+  const { execute: fetch } = useFetch();
```

Comment:
The original reasoning behind the exported method being called `execute` instead of `fetch` was to avoid collision with global, native window objects. We've since decided that calls to window globals should be explicit, IE `window.fetch`. If we find ourselves renaming this to `fetch` frequently, it might be worth changing the name in the hook library itself.